common/export.py

```diff
@@ -381,7 +390,58 @@ def export_json(json_data, filename):
     with open(filename, "w", encoding="utf-8") as file:
         file.write(json_string)
         file.write("\n")  # Add a newline at the end of the file
-    logger.info(f"\nExported {len(json_data)} elements to {filename}")
+
+    logger.info(f"Exported {len(json_data)} elements to {filename}")
+
+
+def format_json(json_path):
+    return subprocess.run(f"npm run fix:prettier {json_path}".split(" "))
+
+
+def export_processes_to_dirs(
+    processes_aggregated_path,
+    processes_impacts_path,
+    processes_corrected_impacts,
+    processes_aggregated_impacts,
+    dirs,
+    extra_data=None,
+    extra_path=None,
+):
+    exported_files = []
+
+    for dir in dirs:
+        logger.info("")
+        logger.info(f"-> Exporting to {dir}")
+        processes_impacts = os.path.join(dir, processes_impacts_path)
+        processes_aggregated = os.path.join(dir, processes_aggregated_path)
+
+        if os.path.isfile(processes_impacts):
+            # Load old processes for comparison
+            oldprocesses = load_json(processes_impacts)
+
+            # Display changes
+            display_changes("id", oldprocesses, processes_corrected_impacts)
+
+        if extra_data is not None and extra_path is not None:
+            extra_file = os.path.join(dir, extra_path)
+            export_json(extra_data, extra_file)
+            exported_files.append(extra_file)
+
+        # Export results
+        export_json(
+            order_json(list(processes_aggregated_impacts.values())), processes_impacts
+        )
```

Comment:
Nit: It feels like we want an `ordered` boolean option available to this `export_json` function